apps/testrunner/test_suites/selenium_tests/main_page_selenium_test.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import logging
+from selenium.webdriver.common.keys import Keys
+from common.mw_url_generator import MWUrlGenerator
+from testrunner.test_suites.selenium_tests.selenium_test import SeleniumTest
+
+logger = logging.getLogger(__name__)
+
+class MainPageSeleniumTest(SeleniumTest):
+    def do_run(self, driver, timer):
```

Comment:
Personally I don't see the reason for this class to exists. There is only one function and it just forwards call to other function...